README.md

```diff
@@ -11,7 +11,10 @@ unpaper
 Status](https://travis-ci.com/unpaper/unpaper.svg?branch=main)](https://travis-ci.com/unpaper/unpaper)
 
 Originally written by Jens Gulden — see AUTHORS for more information.
-Licensed under GNU GPL v2 — see COPYING for more information.
+The entire `unpaper` project is licensed under GNU GPL v2.
+Some of the individual files are licensed under the MIT or Apache 2.0 licenses.
+Each file contains an SPDX license header specifying its license.
```

Comment:
This is great — could I just ask you if you to add a link to [REUSE.software](https://reuse.software/) so that the idea behind is made a bit clearer?

Reply:
Added, what do you think?